markets.json

```diff
@@ -18,13 +18,20 @@
         "symbol": "Bonk",
         "mint": "8LN9tTVkGLqXRT2SxLUjN8YRHiKzYGLq1wGV1FL7E23P",
         "logoUri": "https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/6dhTynDkYsVM7cbF7TKfC9DWB636TcEM935fq7JzL2ES/logo.png"
+      },
+      {
+        "name": "Marinade Staked SOL",
+        "symbol": "mSOL",
+        "mint": "7KowotNADmz2krLdRQKBWygyG4Rsvtne4EG7WU7MBQd8",
+        "logoUri": "https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/So11111111111111111111111111111111111111112/logo.png"
```

Comment:
For consistency, we should make this https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/mSoLzYCxHdYgdzU16g5QSh3i5K3z3KZK7ytfqcJm7So/logo.png

Reply:
good catch ty, fixed